helm/fineract/values.yaml

```diff
@@ -0,0 +1,83 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# TODO: add option to install ActiveMQ?
+
+fineractServer:
+  replicas: 1
+  image:
+    name: apache/fineract
+    tag: latest
+  # key/value pairs to pass as environment variables to the backend pods
+  extraEnv:
+    DRIVERCLASS_NAME: org.drizzle.jdbc.DrizzleDriver
+    PROTOCOL: jdbc
+    SUB_PROTOCOL: mysql:thin
+    fineract_tenants_driver: org.drizzle.jdbc.DrizzleDriver
+    FINERACT_DEFAULT_TENANTDB_PORT: "3306"
+  # sensitive key/value pairs to pass as environment variables to the backend pods as secrets
+  extraSecretEnv: {}
+
+fineractUI:
+  replicas: 1
+  image:
+    name: openmf/community-app
+    tag: latest
+
+# Globals will be available to subcharts
+global:
+  db:
+    tenantsDb: fineract_tenants
+    defaultDb: fineract_default
+
+# Any other parameter from https://artifacthub.io/packages/helm/bitnami/mysql
+mysql:
+  enabled: true
+  # Required because db driver doesn't support mysql 8...
+  image:
+    tag: '5.7'
+  auth:
+    # Please change these...
+    rootPassword: ozIRdODs8Jvs3BzZywgK
+    password: ihcqsCcsX6
```

Comment:
I don't know whether the following line would generate a proper random password. You can try this to generate the mysql password @Yann-J 

`password: {{ randAlphaNum 64 | quote }}`

Or else we can point a particular secret name to the get password. But in order to do that we need to create that secret first before installing helm chart
Refer: https://stackoverflow.com/questions/56170052/how-not-to-overwrite-randomly-generated-secrets-in-helm-templates

Reply:
Hello there, sorry for my delay here!

We can't use `randAlphaNum` inside the chart because that would cause the value to be re-generated each time we deploy. However there's no problem to keep that value empty and implement a runtime warning to instruct users to provide a value.

I'm not so sure that "people will NOT change these" is really true any more with Helm since it's just so easy to do... but for extra protection it makes sense to force it, I can do that.

Reply:
Yeah... I would argue that the situation is a bit different in a Helm context since overriding parameters is dead simple (`helm install --set mysql.auth.password=...`), and really part of the Helm workflow, so the resistance is probably a bit lower there...

Anyway my latest commit is addressing this, and provides an error message in case the values aren't provided.